scanner_test.go

```diff
@@ -93,11 +92,27 @@ func dup(a []*pb.Result) []*pb.Result {
 	return b
 }
 
+func testCallClose(scan *hrpc.Scan, c *mock.MockRPCClient, scannerID uint64,
```

Comment:
you can pass t *testing.T to this function and call t.Helper() at the top of the body. Then handle error from NewScanRange within the function so that you don't need to have those "if" everywhere you call it.

Reply:
Thank you. that is helpful.